remotespark/utils/eventshandler.py

```diff
@@ -0,0 +1,13 @@
+from remotespark.utils.log import Log
+
+
+class EventsHandler:
+    def __init__(self):
+        self.logger = Log("EventsHandler")
+
+    def store_event(self, *args):
```

Comment:
Agreed with @aggFTW 

If the name is `EventsHandler`, it should handle events.  It should store events if its name is `EventStorage`